src/modules/partitionq/CMakeLists.txt

```diff
@@ -0,0 +1,107 @@
+# === This file is part of Calamares - <https://calamares.io> ===
+#
+#   SPDX-FileCopyrightText: 2020 Adriaan de Groot <groot@kde.org>
+#   SPDX-License-Identifier: BSD-2-Clause
+#
+if( NOT WITH_QML )
+    calamares_skip_module( "localeq (QML is not supported in this build)" )
+    return()
```

Comment:
Seems a copy/paste error, should be partitionq, not localeq, copyright header seems to have wrong name, certainly wrong year

Reply:
Fixed in latest commit